### DISCUSSION TITLE
Copilot Fails to Load on VS2022 Community Startup with CompositionFailedException

### DISCUSSION THREAD

**Discussion by geobmx540** [community/community#101189] [Copilot Conversations] [answered]
### Github Copilot Won't load

Getting the following error for VS2022 Community:

### Body

```
GitHub Copilot 1.137.0.0 (v1.137.0.0@2f127d06d)
[Error] An error occurred for 'CopilotPackage': Microsoft.VisualStudio.Composition.CompositionFailedException: Expected 1 export(s) with contract name "GitHub.Copilot.ICopilotClientManager" but found 0 after applying applicable constraints.
   at Microsoft.VisualStudio.Composition.ExportProvider.GetExports(ImportDefinition importDefinition)
   at Microsoft.VisualStudio.Composition.ExportProvider.GetExports[T,TMetadataView](String contractName, ImportCardinality cardinality)
   at Microsoft.VisualStudio.Composition.ExportProvider.GetExport[T,TMetadataView](String contractName)
   at Microsoft.VisualStudio.Composition.ExportProvider.GetExportedValue[T]()
   at Microsoft.VisualStudio.ComponentModelHost.ComponentModel.GetService[T]()
   at GitHub.Copilot.CopilotPackage.<InitializeAsync>d__3.MoveNext() in /_/src/Vsix/CopilotPackage.cs:line 57
 ```

Microsoft Visual Studio Community 2022 (64-bit) - Current
Version 17.8.5
<img width="497" alt="image" src="https://github.com/community/community/assets/662915/01c91326-0a76-476d-9a70-f43e47d9d832">

I've done a few things:
- Updated VS2022
- Updated Github Copilot to latest version
- Run /devenv.exe /clearcache (this this while searching)

**Comment by ayim** [comment]
Hey! First time I'm seeing this specific issue.
Please file a bug using this mechanism within Visual Studio so we can get started at diagnosing.
![image](https://github.com/community/community/assets/10951869/f7d1a09b-57f0-40b6-8dbb-ac96a4e3aaee)


--
Aaron Yim
Product Manager for Copilot in vS

**Reply by geobmx540** [reply]
Hey @ayim - I don't see that "Preview" option, is that a copilot specific thing? 

<img width="214" alt="image" src="https://github.com/community/community/assets/662915/bd60c8c5-ac67-429a-b5e9-7f49b1d0c2f0">

I can use VS search to report a problem:

<img width="599" alt="image" src="https://github.com/community/community/assets/662915/8cef8f23-2532-48a0-9728-2faf69341e62">

Which leads me to the develpercommunity.visualstudio.com website to submit a problem:

<img width="1148" alt="image" src="https://github.com/community/community/assets/662915/b88c1eea-4d4c-4377-b135-5de062b2f490">

If that's the right place, I'll submit a ticket there.

**Reply by ayim** [reply]
That's the right place - I was running a preview build of Visual Studio

**Reply by geobmx540** [reply, marked as answer]
Great - submitted there: https://developercommunity.visualstudio.com/t/Copilot-Fails-to-Load-on-VS2022-Communit/10566963?port=1025&fsid=67b7717a-d2fc-4b3e-b82a-5a3a1bf7df4b

Thanks.
